RCAIDE/Library/Methods/Geometry/Airfoil/import_airfoil_geometry.py

```diff
@@ -0,0 +1,221 @@
+# RCAIDE/Library/Methods/Geometry/Two_Dimensional/Airfoil/import_airfoil_geometry.py
```

Comment:
Don't move this file. Kill it off Racheal has a new version.

Reply:
Done